src/main/java/de/sakpaas/backend/util/RequestUtils.java

```diff
@@ -0,0 +1,52 @@
+package de.sakpaas.backend.util;
+
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import javax.servlet.http.HttpServletRequest;
+import lombok.SneakyThrows;
```

Comment:
Für was brauchst du das?

Reply:
@Mavv3006 `SneakyThrows`? Das unterdrückt eine `Exception` damit mit keine `throws` deklaration an die Methode geschrieben werden muss.

Das ist in diesem Fall in Ordnung, da zwar `MessageDigest.getInstance()` eine `NoSuchAlgorithmException` werfen könnte, allerdings ist in den Specs von Java beschrieben, dass `SHA-256` unterstützt werden muss, also kann dieser Fehler niemals auftreten.

https://docs.oracle.com/javase/8/docs/api/java/security/MessageDigest.html

Reply:
@MrKraboom Dann kann des doch weg, oder?

Reply:
Ne, wenn man `@SneakyThrows` entfernt, muss man ja wieder `throws NoSuchAlgorithmException` an die Methode anfügen und dadurch müssten alle Codestellen die diesen Code verwenden, einen `try-catch`-Block drum machen. Das wollte ich verhindern, indem ich die `Exception` (die eh nie auftreten kann), einfach unterdrücke.